src/core/net/ip6_address.hpp

```diff
@@ -47,6 +47,10 @@
 #include "common/string.hpp"
 #include "mac/mac_types.hpp"
 
+#if OPENTHREAD_CONFIG_BORDER_ROUTING_DHCP6_PD_ENABLE
+#define OTBR_IP6_ADDRESS_SIZE 16
```

Comment:
1. We don't name macros as 'OTBR_xxx' in openthread because there isn't any concept of OTBR from the perspective of openthread.
2. Macros are used in OT C API and C source code (platform implementation). OT core is implemented by cpp and we usually use `constexpr` for constants.
3. Since this definition is only used in `IsRequiredAnyCast`, we can simply add a definition in `IsRequiredAnycast`: `static constexpr uint16_t kIp6AddressSize = 16;`

Reply:
Thanks！